<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayBrowserDialog</class>
 <widget class="QDialog" name="DisplayBrowserDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>276</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Manage Displays</string>
  </property>
  <widget class="Q3ListBox" name="displayListBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>350</width>
     <height>190</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="newButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>221</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">dao/setupdao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
